src/doc.ts

```diff
@@ -0,0 +1,9 @@
+export type Doc = {
+  '@id': string
+};
```

Comment:
I agree. Will take this into account once more things get abstracted away to context.